gytheio-messaging/messaging-broker-activemq/src/test/webapp/admin/WEB-INF/tags/form/short.tag

```diff
@@ -17,8 +17,8 @@
 <%@ attribute name="text" type="java.lang.String" required="true"  %>
 <%@ attribute name="length" type="java.lang.Integer" required="false" %>
 <%
- text = org.apache.commons.lang.StringEscapeUtils.escapeHtml(text);
- text = org.apache.commons.lang.StringEscapeUtils.escapeJavaScript(text);
+ text = org.apache.commons.lang3.StringEscapeUtils.escapeHtml(text);
```

Comment:
Hmm - I wonder if this is correct here (where is this dependency resolved when running the copy of the ActiveMQ "admin" webapp ?

As a separate PR, I wonder if we can remove this test admin webapp (and use docker container instead) ?

Reply:
Jira issue - https://issues.alfresco.com/jira/browse/MM-842